subgraph/src/initializer.ts

```diff
@@ -113,7 +113,7 @@ export function getOrInitMarketEpoch(
     const obj = ipfsJson();
     const speed = fetchDistributionFromDistributionId(
       obj,
-      epochNumber.toString() + "-" + marketSide + "-" + poolTokenAddress.toHexString()
+      "epoch-" + epochNumber.toString() + "-" + marketSide + "-" + poolTokenAddress.toHexString()
```

Comment:
I'm not sure the subgraph is able to handle this, but it would make readability better I think
```suggestion
      ["epoch", epochNumber.toString(), marketSide, poolTokenAddress.toHexString()].join("-");
```